### PR TITLE
Update to Polymer 1.1

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.1.0",
```
Comment (notwaldorf) [inline bower.json]: I don't think this is necessary. `1.1.0` is covered by `^1.0.0` (you can do a `bower cache clean; bower install` to double check

File: bower.json [hunk 2]
```diff
@@ -35,13 +35,13 @@
     "paper-styles": "PolymerElements/paper-styles#^1.0.0"
   },
   "devDependencies": {
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.11",
```
Comment (notwaldorf) [inline bower.json]: `0.7.11` will be picked up by `#^1.0.0`

**Comment by abdonrd** [comment]
Update with the @notwaldorf comments. :smile:

**Comment by abdonrd** [comment]
Update with the @notwaldorf comment in [PolymerElements/paper-button #45](https://github.com/PolymerElements/paper-button/pull/45/files#r37249401).

**Comment by abdonrd** [comment]
Updated with only one commit! :)

**Comment by abdonrd** [comment]
Need more changes, @notwaldorf? :)

**Comment by notwaldorf** [comment]
@abdonrd I am so sorry! I forgot I was supposed to look at these. Thanks for the cleanup! :+1:

**Comment by abdonrd** [comment]
Thank you @notwaldorf! :)

**Pull request merged**